src/account/credentials.cpp

```
@@ -0,0 +1,642 @@
+#include <sys/stat.h>
+#include <sys/types.h>
+#endif
+
+namespace rcli::account {
+namespace {
+
+namespace fs = std::filesystem;
+using Json = nlohmann::json;
+
+constexpr const char* kProductionConsole = "https://console.runanywhere.ai";
+constexpr std::uintmax_t kMaximumCredentialBytes = 1024 * 1024;
```

Clang (14.0.6)

[error] performing an implicit widening conversion to type 'const std::uintmax_t' (aka 'const unsigned long') of a multiplication performed in type 'int'

(bugprone-implicit-widening-of-multiplication-result,-warnings-as-errors)